includes/utils.php

```diff
@@ -159,13 +159,8 @@ function sanitize_credentials( $credentials ) {
  * @return boolean
  */
 function is_indexing() {
```

Comment:
Let's annotate here that this is a "hack" and differs from upstream on purpose, so that the next time we merge this won't get overwritten again.